conan/tools/env/environment.py

```diff
@@ -51,22 +52,23 @@ def environment_wrap_command(env_filenames, env_folder, cmd, subsystem=None,
         raise ConanException("Cannot wrap command with different envs,"
                              "{} - {}".format(bats+ps1s, shs))
 
+    powershell = conanfile.conf.get("tools.env.virtualenv:powershell") or "powershell.exe"
```

Comment:
How is this working for the deprecated syntax? If `"tools.env.virtualenv:powershell` is `True`, wouldn't `powershell` be True and break the launchers?

Reply:
Yes, absolutely, I messed this up with the latest changes, thanks for the comment.